src/acquisition/covidcast/database.py

```diff
@@ -213,10 +213,17 @@ def insert_or_update_batch(self, cc_rows, batch_size=2**20, commit_partial=False
 
 
         result = self._cursor.executemany(insert_into_tmp_sql, args)
+
+        # use the temp table to compute meta just for new data, 
+        # then merge with the global meta
+        meta_update = self.compute_covidcast_meta(table_name=tmp_table_name)
```

Comment:
I'm probably missing somethign really obvious, but where is tmp_table_name set?

Reply:
line 107.  its a part of the normal ingestion process.